static/app/views/automations/components/ruleRow.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/core/button';
+import {IconDelete} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+interface RuleRowProps {
+  children: React.ReactNode;
+  onDelete: () => void;
+}
+
+export default function RuleRow({onDelete, children}: RuleRowProps) {
```

Comment:
Given the change from `RuleNode` -> `DataConditionNode`, should this change from `RuleRow` to `DataConditionRow`? The "Rule" naming feels like a holdover from the old system.

Reply:
yeah i didn't really know what to name this since it's going to be shared between DataConditions and Actions 😓 just `Row` was too generic

Reply:
`AutomationBuilderRow`?

Reply:
renamed in https://github.com/getsentry/sentry/pull/90238/commits/d61c7366ff2e260f47c3f635468f8ceb1178de3c 🫡